src/js/pages/jobs/JobsTable.js

```diff
@@ -42,17 +42,69 @@ class JobsTable extends React.Component {
         prop: 'name',
         render: this.renderHeadline,
         sortable: true,
-        sortFunction: ServiceTableUtil.propCompareFunctionFactory,
+        sortFunction: this.sortJobNames,
         heading
+      }, {
+        className,
+        headerClassName: className,
+        prop: 'status',
+        render: this.renderStatusColumn,
+        sortable: false,
+        heading
+      }, {
+        className,
+        headerClassName: className,
+        prop: 'lastRunStatus',
+        render: this.renderLastRunStatusColumn
       }
     ];
   }
 
+  getData() {
```

Comment:
If we used the structs methods rather than this mapped structure, then we have to do some of the method calls more than once on each render (when sorting especially). I also think it's easier and more human-friendly to look at the `isGroup` property rather than looking at `job instanceof Tree` when needed.

That being said, I don't feel strongly about mapping the data, so if you guys do, I'll be happy to change it.